terrawrap/utils/graph.py

```diff
@@ -135,8 +136,17 @@ def connect_symlinks(graph: networkx.DiGraph, symlink_dict: Dict[str, Set[str]])
         current = symlink
         secondary = 0
         links = list(symlink_dict[symlink])
-        while secondary is not len(links):
+        while secondary != len(links):
             secondary_link = links[secondary]
-            graph.add_edge(current, secondary_link)
+
+            # TODO: implement better way for symlinked tf configs with terraform config
+            try:
+                links_to_run = walk_without_graph_directory(secondary_link)
+                for link in links_to_run:
+                    graph.add_edge(current, link)
+            except ValueError as error:
+                print(error)
+                graph.add_edge(current, secondary_link)
```

Comment:
Why only add secondary_link if there is an exception?

Reply:
Added custom exception for this line.
`graph.add_edge(current, secondary_link)` here to keep previous behavior